jobs/Neatline_Job_DuplicateRecords.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @package     omeka
+ * @subpackage  neatline
+ * @copyright   2014 Rector and Board of Visitors, University of Virginia
+ * @license     http://www.apache.org/licenses/LICENSE-2.0.html
+ */
+
+
+class Neatline_Job_DuplicateRecords extends Omeka_Job_AbstractJob
```

Comment:
Good questions! Some thoughts below on each: 
- Per the discussion on #145, in particular https://github.com/scholarslab/Neatline/issues/145#issuecomment-8135576, I'm envisioning a primary use case where the original exhibit relates to the clone more as a template than as a parent. While there are definitely use cases where it would be helpful to maintain a synchronized relationship between original and clone (and a benefit to storage efficiency), I would think this approach would call for a quasi-version-control system constituting a more involved new feature. 
- Neatline exhibits currently feature dependent delete for records. The effect on memory from the "template" use case is certainly worth further discussing, though, since a typical scenario could involve the number of records being quickly multiplied by the size of a classroom, for instance.
- The existing dependent delete would have to be removed in this case and could be replaced with process called upon exhibit deletion that would check whether any of that exhibit's records are now parentless; this would be straightforward.